comictalker/comictalkerapi.py

```diff
@@ -16,23 +16,50 @@
 from __future__ import annotations
 
 import logging
+import sys
+from typing import NamedTuple
+
+if sys.version_info < (3, 10):
+    from importlib_metadata import EntryPoint, distributions
+else:
+    from importlib.metadata import EntryPoint, distributions
 
-import comictalker.talkers.comicvine
 from comictalker.talkerbase import ComicTalker, TalkerError
 
 logger = logging.getLogger(__name__)
 
 
-def get_comic_talker(source_name: str) -> type[ComicTalker]:
-    """Retrieve the available sources modules"""
-    sources = get_talkers()
-    if source_name not in sources:
-        raise TalkerError(source=source_name, code=4, desc="The talker does not exist")
+class Plugin(NamedTuple):
+    package: str
+    version: str
+    entry_point: EntryPoint
```

Comment:
package and version is not currently used and doesn't work well with what is assigned to it later.